guides/ansible_engine/5-role/README.md

```diff
@@ -0,0 +1,221 @@
+# Exercise 1.5 - Roles: Making your playbooks reusable
+
+While it is possible to write a playbook in one file as we've done throughout this workshop, eventually you’ll want to reuse files and start to organize things.
+
+Ansible Roles is the way we do this.  When you create a role, you deconstruct your playbook into parts and those parts sit in a directory structure.  "Wha??  You mean that seemingly useless [best practice](http://docs.ansible.com/ansible/playbooks_best_practices.html) you mentioned in exercise 1.2?".  Yep, that one.
```

Comment:
"Wha??" Seriously? The whole second part of the paragraph is too informal and needs revision.

Reply:
Haha.. agree, let's clean this up and keep it professional.  A better explanation of roles would be good and or copying from our ansible documentation

Reply:
@liquidat I'd suggest looking at what is in the essentials deck in particular the speaker notes so we are consistent. I also think I pulled those from the docs.